requirements.txt

```diff
@@ -9,4 +9,10 @@ requests
 # For OpenAI GPT
 regex
 # For XLNet
-sentencepiece
\ No newline at end of file
+sentencepiece
+# For XLM
+sacremoses
+pythainlp
```

Comment:
So here we'll only keep `sacremoses` as added dependencies and have detailed error messages for the others